src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -151,31 +151,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
       return;
     }
 
```

Comment:
I tried to use git commit --amend in my local branch (solved merge conflicts locally here) and accidentally ended up with two commits with the same comment now in the remote. Shouldn't be a problem though!